src/components/WelcomeRightContainer/index.tsx

```diff
@@ -0,0 +1,20 @@
+import "./WelcomeRightContainer.scss"
+import mobilePreview from "../../assets/images/i6.png"
+import appstore from "../../assets/images/appstore.png"
+import facebook from "../../assets/images/Facebook.png"
+import twitter from "../../assets/images/Twitter.png"
+
+function WelcomeRightContainer() {
+  return (
+    <section className="right-container">
+      <img className="right-container__mobile-preview" src={mobilePreview} alt="mobile app preview" />
+      <img className="right-container__appstore" src={appstore} alt="app store link" />
+      <div className="right-container__socials">
+        <img className="right-container__socials__facebook" src={facebook} alt="facebook link" />
+        <img className="right-container__socials__twitter" src={twitter} alt="twitter link" />
```

Comment:
In general if using BEM, neither the element not the modifier should reach two levels. Maybe you can follow the block `socials` as you are working inside it, and then go with the elements facebook & twitter, something like `socials-facebook` & `socials-twitter`